src/modules/redesign/i18n/fr_FR/verband/protokoll_types.ts

```diff
@@ -0,0 +1,20 @@
+export default {
+    mission: { regex: /Mission majeur lancée/ },
+    event: { regex: /Évènement d\'alliance lancé/ },
+    appl_accepted: { regex: /Ajouté à l\'alliance/ },
+    appl_declined: { regex: /Refusé à l\'alliance/ },//à vérifié
+    deny_appl: { regex: /Interdit de postuler à l\'alliance/ },
+    allow_appl: { regex: /Autorisé à postuler à l\'alliance/ },
+    left: { regex: /A quitté l\'alliance/ },
+    kicked: { regex: /A été explusé de l\'alliance/ },
```

Comment:
no need to escape the `'` in here. Please remove the `\`. Also the comments (also Line 19) in english please so that we can understand them too ;)

Reply:
ok thank